src/main/models.ts

```diff
@@ -130,6 +130,8 @@ export type FieldError = {
   code: 'missing' | 'invalid'
 }
 
+export type Scopes = string | string[]
```

Comment:
Scope is always singular 😄  (`openid email profile offline_access` is "wider" than `openid profile`)
```suggestion
export type Scope = string | string[]
```